e2e/cypress/integration/enterprise/cloud/billing/subscriptions_free_trial_spec.js

```diff
@@ -0,0 +1,260 @@
+    });
+
+    it('MM-T4129 - Disabling of Subscribe button for not having valid data in Manditory fields in Subscribe screen', () => {
+        // # Click on Subscribe Now button
+        cy.contains('span', 'Subscribe Now').parent().click();
+
+        // # Enter card details
+        getIframeBody().find('[name="cardnumber"]').clear().type('4242424242424242');
+        getIframeBody().find('[name="exp-date"]').clear().type('4242');
+        getIframeBody().find('[name="cvc"]').clear().type('412');
+        cy.get('#input_name').clear().type('test name');
+        cy.contains('legend', 'Country').parent().find('.icon-chevron-down').click();
+        cy.contains('legend', 'Country').parent().find("input[type='text']").type('India{enter}');
+        cy.get('#input_address').type('test1');
+        cy.get('#input_address2').type('test2');
+        cy.get('#input_city').clear().type('testcity');
+        cy.get('#input_state').type('test');
+        cy.get('#input_postalCode').type('444');
+
+        // * Check for enable status of Subscribe button
+        cy.get('.RHS').find('button').should('be.enabled');
+
+        // # Enter invalid csv
+        getIframeBody().find('[name="cvc"]').clear().type('12');
+        cy.get('#input_name').clear().type('test user');
+        cy.get('.RHS').find('button').should('be.disabled');
+
+        // # Enter billing details
+        cy.contains('legend', 'Country').parent().find('.icon-chevron-down').click();
+        cy.contains('legend', 'Country').parent().find("input[type='text']").type('India{enter}');
+        cy.get('.RHS').find('button').should('be.disabled');
+        cy.get('#input_address').type('test1');
+        cy.get('#input_address2').type('test2');
+        cy.get('#input_city').clear().type('testcity');
+        cy.get('.RHS').find('button').should('be.disabled');
+        cy.get('#input_state').type('test');
+        cy.get('.RHS').find('button').should('be.disabled');
+        cy.get('#input_postalCode').type('444');
+
+        //Disable upgrade button when we enter wrong card details
+        getIframeBody().find('[name="cardnumber"]').clear().type('4242424242424141');
+        getIframeBody().find('[name="exp-date"]').clear().type('4242');
+        getIframeBody().find('[name="cvc"]').clear().type('472');
+        cy.get('#input_name').clear().type('test user');
+        cy.get('.RHS').find('button').should('be.disabled');
+    });
+});
+
+const getIframeBody = () => {
+    // # Get the body of iframe element
+    return cy.
+        get('.__PrivateStripeElement > iframe').
+        its('0.contentDocument.body').should('not.be.empty').
+        then(cy.wrap);
+};
+
+const gotoSubscriptionScreen = () => {
+    // # navigating to Subscription Screen
+    cy.get('.sidebar-header-dropdown__icon').click();
+    cy.findByText('System Console').should('be.visible').click();
+    cy.findByText('Subscription').should('be.visible').click();
+};
```

Comment:
This can be removed.

Reply:
gotoSubscriptionScreen has been removed from script